docs/UserGuide.md

```diff
@@ -501,12 +521,22 @@ Format: `remind`
 
 <div class="caption">Reminder window interface.</div>
 
+<br />
+
+[back to table of contents](#)
+
+<br />
+
 <div style="page-break-after: always;"></div>
 
 ### Managing Lesson Fees
 
 This section guides you on how to use the commands for managing the lesson fees of your students in TAB and the behaviour of the Fees Calculator feature of TAB.
 
+1. [Adding a Lesson's Outstanding Fees: `ladd`](#adding-a-lessons-outstanding-fees-ladd)
+2. [Editing a Lesson's Outstanding Fees: `ledit`](#editing-a-lessons-outstanding-fees-ledit)
+3. [Paying a lesson's Outstanding Fees: `paid`](#paying-a-lessons-outstanding-fees-paid)
```

Comment:
Only first word should be caps. Can edit the headings too?
`Behaviour of Fees Calculator` can leave it